usersService/src/main/java/il/ac/afeka/usersservice/logic/ReactiveDepartmentCrud.java

```diff
@@ -0,0 +1,12 @@
+package il.ac.afeka.usersservice.logic;
+
+import il.ac.afeka.usersservice.data.DepartmentEntity;
+import org.springframework.data.mongodb.repository.ReactiveMongoRepository;
+import org.springframework.data.repository.query.Param;
+import reactor.core.publisher.Mono;
+
+public interface ReactiveDepartmentCrud extends ReactiveMongoRepository<DepartmentEntity, String>{
+
+    Mono<DepartmentEntity> findById(@Param("id") String  id);
```

Comment:
I don't know if this is necessary.
What happens if you delete this? does something break?